test/chains.test.ts

```diff
@@ -14,9 +14,7 @@ describe("commands/chains", function () {
   });
 
   test("chains returns correct output", async function () {
-    const chains = Object.fromEntries(
-      Object.entries(getChains())
-    );
+    const chains = Object.fromEntries(Object.entries(getChains()));
```

Comment:
This seems kinda, unnecessary now doesn't it? Can't you just use getChains directly?

Reply:
Yes it does. Updated.